content/en/platform/corda/5.1/application-networks/managing/endpoints/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+date: '2023-09-21'
+title: "Member Endpoints"
+project: corda
+version: 'Corda 5.1'
+menu:
+  corda51:
+    identifier: corda51-networks-member-endpoints
+    parent: corda51-networks-manage
+    weight: 3000
+section_menu: corda51
+---
+# Member Endpoints
+
+If a member needs to make changes to their endpoint information, for example, changing the URL of the endpoint or adding a new endpoint, the member must re-register with the updated endpoint information. To minimize communication disruption, the member should re-register with both the old and new information, before removing the old information, as follows:
```

Comment:
We don't really need the member to re-register with old and new information, we only recommend both endpoints be added to the Gateway config. The member-provided context during re-registration only contains the new endpoint (step 3)